src/NHibernate/Impl/SessionImpl.cs

```diff
@@ -1859,23 +1870,10 @@ public override void List(CriteriaImpl criteria, IList results)
 
 				string[] implementors = Factory.GetImplementors(criteria.EntityOrClassName);
 				int size = implementors.Length;
-
```

Comment:
It's not a problem, I just don't see what is the point of this irrelevant change.